src/main/scala/is/hail/expr/ir/functions/MathFunctions.scala

```diff
@@ -22,6 +22,20 @@ object MathFunctions extends RegistryFunctions {
 
   def ceil(x: Double): Double = math.ceil(x)
 
+  def mod(x: Float, y: Float): Float = {
```

Comment:
added some more "%" and "**" functions and made the types that they take the same as the function registry